src/Checkout/Document/Zugferd/ZugferdSubscriber.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\Checkout\Document\Zugferd;
+
+use Shopware\Core\Checkout\Document\Zugferd\ZugferdInvoiceGeneratedEvent;
+use Shopware\Core\Framework\Log\Package;
+use Swag\PayPal\SwagPayPal;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * @internal
+ */
+#[Package('checkout')]
+class ZugferdSubscriber implements EventSubscriberInterface
+{
+    public function __construct(
+        private readonly TranslatorInterface $translator
+    ) {
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        if (!\class_exists(ZugferdInvoiceGeneratedEvent::class)) {
+            return [];
+        }
+
+        return [
+            ZugferdInvoiceGeneratedEvent::class => 'generateInvoice',
+        ];
+    }
+
+    public function generateInvoice(ZugferdInvoiceGeneratedEvent $event): void
+    {
+        // Method added with v6.6.10.6
+        // @phpstan-ignore-next-line function.alreadyNarrowedType
```

Comment:
```suggestion
        // @phpstan-ignore function.alreadyNarrowedType
```

ignoring with identifier only works with `@phpstan-ignore`